spec/formatting.md

```diff
@@ -942,9 +942,9 @@ The _Default Bidi Strategy_ is defined as follows:
       True if the `u:dir` _option_ of the _resolved value_ of `exp` has a value other than `'inherit'`,
       or False otherwise.
    1. If `dir` is `'LTR'`:
-      1. If `msgdir` is `'LTR'` in the formatted output
+      1. If `msgdir` is `'LTR'`
```

Comment:
I think this is a good change, in that it is clearer. `msgdir` is whatever it is.